front/.eslintrc.json

```diff
@@ -24,7 +24,6 @@
         "@typescript-eslint"
     ],
     "rules": {
-        "no-unused-vars": "off",
-        "@typescript-eslint/no-explicit-any": "error"
```

Comment:
I'd really prefer forbidding explicit anys as much as possible. They are a code smell, they make Typescript 100% useless. It is in my opinion the worst feature of Typescript.

Are you facing an issue with a plugin not providing typings?
You could simply add a comment to disable the "any" check for this specific line?

Reply:
There is no point in using a es-lint rule if we regulary ignore it, which we are doing already, and will continue doing.
None of the rex plugins are typed, which means that correct way to handle this is to use the 'any' type. 

'any' isn't a code smell when we have to deal with libraries that aren't typed and it is definitly better to accept it rather than pretend we don't need to use it.

Reply:
any is not a code smell - explicitly typing something as any is
the correct way to handle untyped libraries is to type them as much as needed